infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -907,7 +908,9 @@ private LogEntry getLogEntry(long address, LogData entry) {
             }
 
             allRecordsBuf.flip();
-            writeByteBuffer(segment.getWriteChannel(), allRecordsBuf);
+            // This API is called by stateTransfer, is that considered a high priority
+            // operation
+            writeByteBuffer(segment.getWriteChannel(), allRecordsBuf, true);
```

Comment:
The comment says if checkQuota is false then it is a high priority request, but here true is passed instead?
To avoid confusion, should it just pass Priority itself instead of binding it to yet another decision variable?

Reply:
Good suggestion. Done.